app.rb

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env ruby
+
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+require_relative './classroom'
+
+class App
+  attr_accessor :books, :people, :rentals, :classroom
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @classroom = ['Full-stack Development']
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "Name: \"#{person.name}\", Age: #{person.age}, ID: #{person.id}, Class: #{person.class}"
+    end
+  end
+
+  def create_person
+    puts 'What do you want to create, a Teacher (T) or a Student (S)? [Enter the Letter]:'
+    input_type = gets.chomp.upcase
+    case input_type
+    when 'T'
+      create_teacher
+    when 'S'
+      create_student
+    else
```

Comment:
- [ ] When I try to create a teacher or student, the app crashes with the errors shown in the screenshot below. Kindly take a look at the arguments passed and fix the issue so a person can be created in order for the rental feature to work 👍🏽

<img width="1255" alt="Screenshot 2023-03-17 at 10 09 30 PM" src="https://user-images.githubusercontent.com/50941074/226061902-c449a528-fd4b-46c5-ba20-9b2b5bd7ea3f.png">

Reply:
- Well resolved @yetemegn-telaye 👍🏽